### DISCUSSION TITLE
ART 1.11.0

### DISCUSSION THREAD

**Discussion by beat-buesser** [Trusted-AI/adversarial-robustness-toolbox#1765] [Announcements]
This release of ART 1.11.0 introduces estimators for YOLO object detection and regression models, the first audio poisoning attack, new quey-efficient black-box evasion attacks, certified defenses against adversarial patch attacks, metrics quantifying membership inference and more. 

# Added

- Added Momentum-Iterative FGSM evasion attack in `MomentumIterativeMethod` and added optional momentum to loss gradients in `ProjectedGradientDescent*` attacks. (#1614)
- Added metrics measuring worst-case scores of membership inference attacks. (#1709)
- Added estimator for YOLO v3 models in PyTorch in `PyTorchYolo`. (#1715)
- Added estimators for de-randomized smoothing certification against patch attacks in `PyTorchDeRandomizedSmoothing` and `TensorFlowV2DeRandomizedSmoothing`. (#1729)
- Added query-efficient hard-label black-box evasion attack Sign-Opt in `SignOPTAttack`. (#1730)
- Added Sleeper Agent poisoning attack PyTorch in `SleeperAgentAttack`. (#1736)
- Added exclusionary reclassification to `ActivationDefence`. (#1738)
- Added dirty-label backdoor poisoning attack on audio classification in `art.attacks.poisoning.perturbations.audio_perturbations`. (#1740)
- Added estimators for regression in `PyTorchRegressor` and `KerasRegressor` for PyTorch and Keras. (#1651)
- Added option for targeted attacks to `AdversarialPatch` and `AdversarialPatchNumpy`. (#1759)

# Changed

- Changed `check_and_transform_label_format` for `nb_classes=None` to automatically determine the number of classes in the provided labels. (#1747)
- Added additional documentation to `ZOOAttack` and cleaned up the code of method `compare`. (#1648)
- Changed default value for number of epochs `nb_epochs` in `AdversarialTrainerMadryPGD` to match 80'000 training steps of Madry et al. (#1758)

# Removed

[None] 

# Fixed

- Fixed `PyTorchClassifier.clone_fore_refitting` by deleting optimizer from parameters before calling `set_param()` to avoid creating the cloned model with the old optimizer. (#1742)
- Fixed missing propagation of `nb_classes` to method `check_and_transform_label_format` in inference attacks. (#1713)

<hr /><em>This discussion was created from the release <a href='https://github.com/Trusted-AI/adversarial-robustness-toolbox/releases/tag/1.11.0'>ART 1.11.0</a>.</em>
